src/components/TooltipLearnMoreModal/index.tsx

```diff
@@ -66,34 +67,36 @@ export const LearnMoreLink = (props: { onClick: () => void }): JSX.Element => {
   );
 };
 
-export const LearnMoreModalContentGrowthForm = (): JSX.Element => (
-  <>
-    <p>
-      <strong>{strings.FERN}:</strong> {strings.LEARN_MORE_GROWTH_FORM_FERN}
-    </p>
-    <p>
-      <strong>{strings.GRAMINOID}:</strong> {strings.LEARN_MORE_GROWTH_FORM_GRAMINOID}
-    </p>
-    <p>
-      <strong>{strings.FORB}:</strong> {strings.LEARN_MORE_GROWTH_FORM_FORB}
-    </p>
-    <p>
-      <strong>{strings.SHRUB}:</strong> {strings.LEARN_MORE_GROWTH_FORM_SHRUB}
-    </p>
-    <p>
-      <strong>{strings.TREE}:</strong> {strings.LEARN_MORE_GROWTH_FORM_TREE}
-    </p>
-    <p>
-      <strong>{strings.FUNGUS}:</strong> {strings.LEARN_MORE_GROWTH_FORM_FUNGUS}
-    </p>
-    <p>
-      <strong>{strings.LICHEN}:</strong> {strings.LEARN_MORE_GROWTH_FORM_LICHEN}
-    </p>
-    <p>
-      <strong>{strings.MOSS}:</strong> {strings.LEARN_MORE_GROWTH_FORM_MOSS}
-    </p>
-  </>
-);
+export const LearnMoreModalContentGrowthForm = (): JSX.Element => {
+  const { activeLocale } = useLocalization();
+  const collator = new Intl.Collator(activeLocale || undefined);
+
+  const elements: JSX.Element[] = [
+    [strings.FERN, strings.LEARN_MORE_GROWTH_FORM_FERN],
+    [strings.GRAMINOID, strings.LEARN_MORE_GROWTH_FORM_GRAMINOID],
+    [strings.FORB, strings.LEARN_MORE_GROWTH_FORM_FORB],
+    [strings.SHRUB, strings.LEARN_MORE_GROWTH_FORM_SHRUB],
+    [strings.TREE, strings.LEARN_MORE_GROWTH_FORM_TREE],
+    [strings.FUNGUS, strings.LEARN_MORE_GROWTH_FORM_FUNGUS],
+    [strings.LICHEN, strings.LEARN_MORE_GROWTH_FORM_LICHEN],
+    [strings.MOSS, strings.LEARN_MORE_GROWTH_FORM_MOSS],
+    [strings.VINE, strings.LEARN_MORE_GROWTH_FORM_VINE],
+    [strings.LIANA, strings.LEARN_MORE_GROWTH_FORM_LIANA],
+    [strings.SHRUB_TREE, strings.LEARN_MORE_GROWTH_FORM_SHRUB_TREE],
+    [strings.SUBSHRUB, strings.LEARN_MORE_GROWTH_FORM_SUBSHRUB],
+    [strings.MULTIPLE_FORMS, strings.LEARN_MORE_GROWTH_FORM_MULTIPLE_FORMS],
+  ]
+    .sort((a, b) => collator.compare(a[0], b[0]))
+    .map((x) => (
+      <>
```

Comment:
if we want the same structure as before we don't need the` < >  < / > ` here.

Reply:
Without those, I get an error:
```
ERROR: 91:9  jsx-key  Missing "key" prop for element.
```
Is there a better way to solve that?

Reply:
Yes, you can add a key prop to the <p> like this:
`<p key={`growthForm-${index}`}>` 
and to have that index available, in line 90, in the map function add the index as parameter:
`    .map((x, index) => (`